big_tests/tests/inbox_SUITE.erl

```diff
@@ -761,9 +802,270 @@ groupchat_markers_all_reset_room_created(Config) ->
+    wait_for_groupchat_msg(Users),
+
+    [AliceJid, BobJid, KateJid] = lists:map(fun to_bare_lower/1, Users),
+    BobRoomJid = muc_room_address(Room, nick(Bob)),
+    %% Bob has 0 unread messages
+    check_inbox(Bob, #inbox{
+      total = 1,
+      convs = [#conv{unread = 0, from = BobRoomJid, to = BobJid,
+                     content = Msg}]}, #{case_sensitive => true}),
+    %% Alice have one conv with 1 unread message
+    check_inbox(Alice, #inbox{
+      total = 1,
+      convs = [#conv{unread = 1, from = BobRoomJid, to = AliceJid, content = Msg}]}, #{case_sensitive => true}),
+    %% Kate has 0 unread messages
+    check_inbox(Kate, #inbox{
+      total = 1,
+      convs = [#conv{unread = 0, from = BobRoomJid, to = KateJid, content = Msg}]}, #{case_sensitive => true})
+    end).
+
+private_messages_are_handled_as_one2one(Config) ->
+  escalus:story(Config, [{alice, 1}, {bob, 1}, {kate, 1}], fun(Alice, Bob, Kate) ->
+    Users = [Alice, Bob, Kate],
+    Msg = <<"Hi Room!">>,
+    Id = <<"MyID">>,
+    Room = ?config(room, Config),
+    RoomAddr = muc_room_address(Room),
+    BobRoomJid = muc_room_address(Room, nick(Bob)),
+    KateRoomJid = muc_room_address(Room, nick(Kate)),
+
+    enter_room(Room, Users),
+    make_members(Room, Alice, Users -- [Alice]),
+    Stanza = escalus_stanza:set_id(
+      escalus_stanza:chat_to(BobRoomJid, Msg), Id),
+    escalus:send(Kate, Stanza),
+    BobsPrivMsg = escalus:wait_for_stanza(Bob),
+
+    [_AliceJid, BobJid, KateJid] = lists:map(fun to_bare_lower/1, Users),
+    %% Bob has 1 unread message
+    check_inbox(Bob, #inbox{
+      total = 1,
+      convs = [#conv{unread = 1, from = KateRoomJid, to = BobRoomJid,
+                     content = Msg}]}, #{case_sensitive => true}),
+    %% Alice gets nothing
+    check_inbox(Alice, #inbox{
+      total = 0,
+      convs = []}),
+    %% Kate has 1 conv with 0 unread messages
+    check_inbox(Kate, #inbox{
+      total = 1,
+      convs = [#conv{unread = 0, from = KateJid, to = BobRoomJid, content = Msg}]},
+                #{case_sensitive => true, check_resource => false})
+    end).
+
+
+
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% Helpers
 
+go_offline(User, Room, Occupants) ->
+    UnavailavbleStanza = escalus_stanza:presence(<<"unavailable">>),
+    Stanza = muc_helper:stanza_to_room(UnavailavbleStanza, Room, nick(User)),
+    escalus:send(User, Stanza),
+    lists:foreach(fun(User) -> A = escalus:wait_for_stanza(User), ct:pal("A: ~p", [A]) end,
+                  Occupants).
+
+go_online(User, Room, Occupants) ->
+    UnavailavbleStanza = escalus_stanza:presence(<<"unavailable">>),
+    Stanza = muc_helper:stanza_to_room(UnavailavbleStanza, Room, nick(User)),
+    escalus:send(User, Stanza),
+    lists:foreach(fun(User) -> A = escalus:wait_for_stanza(User), ct:pal("A: ~p", [A]) end,
+                  Occupants).
+
+wait_for_groupchat_msg(Users) ->
+    Resps = lists:map(fun(User) -> escalus:wait_for_stanza(User) end,
+              Users),
```

Comment:
The author of the suite used two spaces instead of 4. I planned to change the indentation in the whole file in the very last commit of this PR so that it doesn't hide the real changes.